x/nexus/types/proposal.go

```diff
@@ -0,0 +1,95 @@
+package types
+
+import (
+	"encoding/hex"
+	fmt "fmt"
+	"strings"
+
+	gov "github.com/cosmos/cosmos-sdk/x/gov/types"
+
+	"github.com/axelarnetwork/axelar-core/utils"
+	"github.com/axelarnetwork/utils/slices"
+)
+
+const (
+	ProposalTypeCallContracts string = "CallContracts"
+)
+
+func init() {
+	gov.RegisterProposalType(ProposalTypeCallContracts)
```

Comment:
```suggestion
	gov.RegisterProposalType(ProposalTypeCallContracts)
```
Can you also register the amino codec?
`gov.RegisterProposalTypeCodec(&CallContractsProposal{}, "axelar/CallContractsProposal")`

Signing from ledger requires amino for e.g., and given the high deposit amount this scenario might be practical